build/nested-cluster/start.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+wait_for_docker ()
+{
+  # Wait for docker.
+  until docker version; do sleep 1 ;done
+}
+
+start_kubelet ()
+{
+  # Start the kubelet.
+  mkdir -p /etc/kubernetes/manifests
+  mkdir -p /etc/srv/kubernetes
+
+  # Change the kubelet to not fail with swap on.
+  sed -i 's/ExecStart=\/usr\/bin\/kubelet/ExecStart=\/usr\/bin\/kubelet --fail-swap-on=false/' /etc/systemd/system/kubelet.service.d/kubeadm-10.conf
```

Comment:
that should be 10-kubeadm.conf (did it change?)
for robustness it might be better to generate a `20-something.conf` instead, that would set `KUBELET_EXTRA_ARGS`

Reply:
Put the extra args in a separate file. It's better.